riposte-spi/src/main/java/com/nike/riposte/server/http/RequestInfo.java

```diff
@@ -384,4 +384,15 @@ public default String getPathParam(String key) {
      * do not need to worry about this issue - it's a problem for the server to solve.
      */
     public void releaseMultipartData();
+
+    /**
+     * This will return the path template set by {@link #setPathParamsBasedOnPathTemplate(String)} .
```

Comment:
thanks for the recommendations. i'll keep that formatting in mind for future comments. 

I've updated the PR with your suggestions. let me know if you think of other wording changes.